### ISSUE TITLE
Wait for some tasks but not for others

### ISSUE THREAD

**Issue by danielru** [IBM/torc_py#2]
I was wondering if/how it is possible, to wait for only the completion of some sub-tasks but not all. If I am reading the code correct, the `torc.wait()` function does not use arguments, is that right?

I was hoping to do something along the lines of

`task1 = torc.submit(...)`. 
`task2 = torc.submit(...)`. 
`torc.wait(task1)`

so that the code only has to wait for `task1` to complete but can continue while `task2` is still running. Is that possible?

**Comment by phadjido** [comment]
Yes, torc.wait actually works as torc.waitall, mainly for simplicity in the interface (API) and the implementation. 
Partial wait of tasks can be supported with some code additions, I can try to add it. 
My only concern, from a user perspective, is the management (waiting) of the rest of the submitted tasks. To keep the API simple, all the pending child tasks (if any) should be waited at some point.

**Comment by phadjido** [comment]
It has not been tested thoroughly yet, but the new branch `waitsome` provides support for partial wait of tasks, with small internal code additions. 
Example code: https://github.com/IBM/torc_py/blob/waitsome/examples/ex12_waitsome.py

**Comment by danielru** [comment]
@phadjido Thanks a lot for the quick reply! I'll give it a go and let you know if I encounter any problems.
